src/applications/disability-benefits/all-claims/pages/addConditionsPages.js

```diff
@@ -0,0 +1,373 @@
+      charcount: true,
+    }),
+    worsenedDescription: textUI({
+      title:
+        'Please briefly describe the injury or exposure during your military service that caused your existing disability to get worse.',
+      hideIf: (formData, index) =>
+        createCauseFollowUpConditional(formData, index, 'WORSENED'),
+      required: (formData, index) =>
+        !createCauseFollowUpConditional(formData, index, 'WORSENED'),
+      charcount: true,
+    }),
+    worsenedEffects: textareaUI({
+      title:
+        'Please tell us how the disability affected you before your service, and how it affects you now after your service.',
+      hideIf: (formData, index) =>
+        createCauseFollowUpConditional(formData, index, 'WORSENED'),
+      required: (formData, index) =>
+        !createCauseFollowUpConditional(formData, index, 'WORSENED'),
+      charcount: true,
+    }),
+    vaMistreatmentDescription: textareaUI({
+      title:
+        'Please briefly describe the injury or event while you were under VA care that caused your disability.',
+      hideIf: (formData, index) =>
+        createCauseFollowUpConditional(formData, index, 'VA'),
+      required: (formData, index) =>
+        !createCauseFollowUpConditional(formData, index, 'VA'),
+      charcount: true,
+    }),
+    vaMistreatmentLocation: textUI({
+      title: 'Please tell us where this happened.',
+      hideIf: (formData, index) =>
+        createCauseFollowUpConditional(formData, index, 'VA'),
+      required: (formData, index) =>
+        !createCauseFollowUpConditional(formData, index, 'VA'),
+      charcount: true,
+    }),
+    vaMistreatmentDate: currentOrPastMonthYearDateUI({
+      title:
+        'Please tell us when this happened. If you’re having trouble remembering, do your best to estimate.',
+      hideIf: (formData, index) =>
+        createCauseFollowUpConditional(formData, index, 'VA'),
+      required: (formData, index) =>
+        !createCauseFollowUpConditional(formData, index, 'VA'),
+    }),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      primaryDescription: {
+        type: 'string',
+        maxLength: 400,
+      },
+      causedByCondition: selectSchema(Object.values(disabilityLabelsRevised)),
+      causedByConditionDescription: {
+        type: 'string',
+        maxLength: 400,
+      },
+      worsenedDescription: {
+        type: 'string',
+        maxLength: 50,
+      },
+      worsenedEffects: {
+        type: 'string',
+        maxLength: 350,
+      },
+      vaMistreatmentDescription: {
+        type: 'string',
+        maxLength: 350,
+      },
+      vaMistreatmentLocation: {
+        type: 'string',
+        maxLength: 25,
+      },
+      vaMistreatmentDate: currentOrPastMonthYearDateSchema,
+    },
+  },
+};
+
+/**
+ * This page is skipped on the first loop for required flow
+ * Cards are populated on this page above the uiSchema if items are present
+ *
+ * @returns {PageSchema}
+ */
+const summaryPage = {
```

Comment:
For the Edit button it announces as "Edit edit `condition`". Delete button is okay.
<img width="926" alt="Screenshot 2024-11-03 at 11 39 11 AM" src="https://github.com/user-attachments/assets/668601da-26e0-4c8f-a228-8b896e7b109f">